notebooks/ifu_optimal/ifu_optimal.ipynb

```diff
@@ -4,18 +4,22 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #32.        data_xrange=[6,33]
</pre><p>@PatrickOgle, Did you mean add <code>data_yrange</code> here instead? I see both are set for <code>data_xrange</code>. I think that's a typo.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jdat_notebooks/pull/191/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>